subsys/bluetooth/host/gatt.c

```diff
@@ -2314,6 +2314,7 @@ static void gatt_find_type_rsp(struct bt_conn *conn, u8_t err,
 	     i++, length -=  sizeof(rsp->list[i])) {
 		struct bt_gatt_attr attr = {};
 		struct bt_gatt_service_val value;
+		struct bt_uuid_16 uuid_svc;
```

Comment:
nitpick, do we still follow reverse Christmas tree declaration coding style?

Reply:
We do, would be good if @joerchan placed this at the top.

Reply:
Done.